openedx/adg/lms/applications/managers.py

```diff
@@ -29,6 +31,56 @@ def prereq_course_groups(self):
         """
         return self.get_queryset().filter(is_program_prerequisite=True, multilingual_courses__isnull=False).distinct()
 
+    def get_user_program_prereq_courses(self, user):
+        """
+        Returns program prerequisite courses for a user.
+        Args:
+            user (User): User for which program prerequisite courses will be returned
+        Returns:
+            list: List of program prerequisite courses for a user
+        """
+        return get_courses_from_course_groups(
+            self.prereq_course_groups(), user
+        )
+
+    def business_line_and_common_business_line_prereq_course_groups(self):
+        """
+        Returns prerequisite course groups for the selected business line and common prerequisite course groups
+        for all business lines.
+        Returns:
+            list: List of business line prerequisite course groups for a user
+        """
```

Comment:
Lets follow the convention and add spaces between function definition, Args and Returns